stairsplus/registrations.lua

```diff
@@ -74,3 +74,40 @@ for _, name in pairs(default_nodes) do
 		})
 	end
 end
+
+-- wool registrations
+
+if minetest.get_modpath("wool") then
+
+	local colorlist = {
+		{"white",      "White"},
+		{"grey",       "Grey"},
+		{"black",      "Black"},
+		{"red",        "Red"},
+		{"yellow",     "Yellow"},
+		{"green",      "Green"},
+		{"cyan",       "Cyan"},
+		{"blue",       "Blue"},
+		{"magenta",    "Magenta"},
+		{"orange",     "Orange"},
+		{"violet",     "Violet"},
+		{"brown",      "Brown"},
+		{"pink",       "Pink"},
+		{"dark_grey",  "Dark Grey"},
+		{"dark_green", "Dark Green"},
+	}
+
+	for i in ipairs(colorlist) do
```

Comment:
@kilbith The idea behind having explicit descriptions is to make I18N easier. Generating descriptions that way makes it harder. See my other comment.